src/views/download-view.js

```diff
@@ -134,7 +134,7 @@ class DownloadView extends PolymerElement {
                          </span>
                     </p>
 
-                    If you are looking for older versions please check out our
+                    If you are looking for the latest version please check out our
```

Comment:
I think the latest version is already listed in version. Did you mean "nightly version"? I think we should remove this hint completely

Reply:
Latest version is correct. The latest version is listed on github first, before anywhere else, especially our documentation ;)

Alas i agree with your comment, might as well remove it.